### PR TITLE
chore: migrate connector node

### PR DESCRIPTION
**Pull request by tabVersion**
Signed-off-by: tabVersion <tabvision@bupt.icu>I hereby agree to the terms of the [RisingWave Labs, Inc. Contributor License Agreement](https://gist.github.com/TennyZhuang/f00be7f16996ea48effb049aa7be4d66#file-rw_cla).

## Checklist For Contributors
~~- [ ] I have written necessary rustdoc comments~~
~~- [ ] I have added necessary unit tests and integration tests~~
~~- [ ] I have added fuzzing tests or opened an issue to track them. (Optional, recommended for new SQL features #7934).~~
~~- [ ] I have demonstrated that backward compatibility is not broken by breaking changes and created issues to track deprecated features to be removed in the future. (Please refer to the issue)~~
~~- [ ] All checks passed in `./risedev check` (or alias, `./risedev c`)~~

## Checklist For Reviewers

- [x] I have requested macro/micro-benchmarks as this PR can affect performance substantially, and the results are shown.

## Documentation

- [x] My PR **DOES NOT** contain user-facing changes.

<details><summary>Click here for Documentation</summary>

### Types of user-facing changes

Please keep the types that apply to your changes, and remove the others.

- Installation and deployment
- Connector (sources & sinks)
- SQL commands, functions, and operators
- RisingWave cluster configuration changes
- Other (please specify in the release note below)

### Release note

</details>

### CODE REVIEW

**Review by huangjw806 — COMMENTED** [review]
It seems that the trigger conditions of the four workflows are the same. I wonder if they can be combined into one yaml file?

**Comment by tabVersion** [comment]
> It seems that the trigger conditions of the four workflows are the same. I wonder if they can be combined into one yaml file?

fixed

**Review by huangjw806 — APPROVED** [review]
LGTM!

File: .github/workflows/connector-node-integration.yml [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+      - name: Set up JDK 11
+        uses: actions/setup-java@v3
+        with:
+          java-version: '11'
+          distribution: 'adopt'
+          cache: 'maven'
+      - name: run integration tests
+        run: |
+          cd connector_node
+          mvn --batch-mode --update-snapshots clean package  -DskipTests=true
```
Comment (StrikeW) [inline .github/workflows/connector-node-integration.yml:26]: Do we plan to migrate to BuildKite or still maintain this GitHub action workflow? I am ok with using GitHub action for integration tests of sinks.  cc @huangjw806

**Review by StrikeW — APPROVED** [review]
LGTM

**Comment by BugenZhao** [comment]
Is it possible to avoid running connector tests if there's no change to `./connector_node`?

File: .github/workflows/connector-node-integration.yml (hunk 1 above)
Comment (tabVersion) [reply .github/workflows/connector-node-integration.yml:26]: it is a temp solution. migrate ci later

**Comment by tabVersion** [comment]
> Is it possible to avoid running connector tests if there's no change to `./connector_node`?

cc @huangjw806 . same question. if no edit to `./proto` and `./connector_node`, the this part can be skipped

File: .github/workflows/connector-node-integration.yml [hunk 2]
```diff
@@ -0,0 +1,130 @@
+name: Connector Node Integration tests
+
+on:
```
Comment (huangjw806) [inline .github/workflows/connector-node-integration.yml:3]: Perhaps a condition should be added here, which will only be triggered when the connector-node changes.

**Pull request merged**